force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -121,8 +178,14 @@ public with sharing class VariablesDataTypesOperators {
 	 * @return The complete sentence as a single String.
 	 */
 	public static String createSentence(String noun, String verb, String endingPunctuation) {
-		return null; // Replace null with the variable you used to store the result
+		// Check for null inputs
+        if (noun == null || verb == null || endingPunctuation == null) {
```

Comment:
You want to be careful with the 'OR'. If only the endingPunctuation is null and the noun and verb are not null, should you return null?